### PR TITLE
Move config handling to common module

### PR DESCRIPTION
**Pull request by gingermusketeer**
This builds on https://github.com/eik-lib/common/pull/84.

Moving this to the common module means we can take advantage of the new checks to ensure that the configured files exist on the filesystem.

Further changes could be made to make better use of the EikConfig object being passed around but thought that this was a good start.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@babel/core": "^7.12.3",
     "@babel/preset-env": "^7.12.1",
-    "@eik/common": "^1.3.0",
+    "@eik/common": "git+ssh://git@github.com/gingermusketeer/common.git#add-common-config-handling-from-cli",
```
Comment (digitalsadhu) [inline package.json:30]: change back to 1.4.0 which is now released

**Review by digitalsadhu — APPROVED** [review]
Looking sharp, nice improvement LGTM

**Comment by gingermusketeer** [comment]
Tests don't seem to be running on this branch?

**Comment by digitalsadhu** [comment]
no... they don't do they

**Pull request merged**